apps/frontend-pwa/src/pages/index.tsx

```diff
@@ -52,6 +54,8 @@ const Index = function () {
   }, [])
 
   const [subscribeToPush] = useMutation(SubscribeToPushDocument)
+  const [unsubscribeFromPush] = useMutation(UnsubscribeFromPushDocument)
+  const [sendPushNotifications] = useMutation(SendPushNotificationsDocument)
```

Comment:
this should be removed before the merge, as we do not want to allow triggering push from frontend